lib/src/scenario/TestScenarioFactory.ts

```
@@ -0,0 +1,600 @@
+        baseScenario.name,
+        baseScenario.organization
+      );
+      baseScenario.scenarioSetupSucceeded = true;
+
+      LogManager.getLogger().info("Initial base scenario setup created");
+      if (!scenarioConfig.space) {
+        // nothing more to do, return
+        return baseScenario;
+      }
+
+      baseScenario.space = await this.createRootSpace(
```

Biome (1.9.4)

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)